src/main/java/macrobase/conf/MacroBaseConf.java

```diff
@@ -166,6 +170,9 @@ public BatchTrainScore constructTransform (TransformType transformType)
             case BAYESIAN_NORMAL:
                 log.info("Using Bayesian Normal transform.");
                 return new BayesianNormalDensity(this);
+            case GAUSSIAN_MIXTURE_EM:
```

Comment:
Is it worth calling this out as a finite gaussian mixture? Put another way, what will the nonparametric estimator look like?